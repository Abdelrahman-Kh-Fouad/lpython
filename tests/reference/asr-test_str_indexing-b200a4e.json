{
    "basename": "asr-test_str_indexing-b200a4e",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_str_indexing.py",
    "infile_hash": "d237030704fd7f04d4cad59083ca0d663561f9be6f3407e3ed04ed78",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-test_str_indexing-b200a4e.stdout",
    "stdout_hash": "12c1ab98ba43be8e9d1d10d1ff7800868c7161d2d374ed0bfd8b605f",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}