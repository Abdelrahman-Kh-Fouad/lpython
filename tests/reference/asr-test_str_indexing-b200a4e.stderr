semantic error: string indices must be integers
 --> tests/errors/test_str_indexing.py:4:13
  |
4 |     print(s[1.2])
  |             ^^^ 
